extensions/amp-story/1.0/page-advancement.js

```diff
@@ -546,16 +546,18 @@ export class ManualAdvancement extends AdvancementConfig {
   }
 
   /**
-   * Checks if click was inside of one of the side edges of the screen.
+   * Checks if click was inside of one of the side edges of the page.
    * @param {!Event} event
    * @param {!ClientRect} pageRect
    * @return {boolean}
    * @private
    */
-  isInScreenSideEdge_(event, pageRect) {
+  isInStoryPageSideEdge_(event, pageRect) {
```

Comment:
I think you can remove `pageRect` here.

Reply:
Changed this bc offsetLeft & Top just weren't working on loading pages.